docs/api-references.md

```diff
@@ -0,0 +1,1373 @@
+# API Reference
+
+## Packages
+- [cluster.kubernetes-fleet.io/v1beta1](#clusterkubernetes-fleetiov1beta1)
+- [fleet.azure.com/v1alpha1](#fleetazurecomv1alpha1)
+- [placement.kubernetes-fleet.io/v1beta1](#placementkubernetes-fleetiov1beta1)
+
+
+## cluster.kubernetes-fleet.io/v1beta1
+
+
+
+### Resource Types
+- [InternalMemberCluster](#internalmembercluster)
```

Comment:
please only include external facing API

Reply:
Sure👌

Reply:
I will hide `InternalMemberCluster` and `AppliedWorks`.

Reply:
Also removed reference to v1alpha1 APIs.